src/ext/dispatch_compat_php5.c

```diff
@@ -382,4 +390,12 @@ void ddtrace_execute_tracing_closure(zval *callable, zval *span_data, zend_execu
     }
     zend_fcall_info_args_clear(&fci, 0);
 }
+
+void ddtrace_span_attach_exception(ddtrace_span_t *span, zval *exception) {
+    if (exception) {
```

Comment:
would it make sense to check  if `Z_TYPE_P(tmp) != IS_NULL)`

Reply:
IIRC it's always a null pointer or an object. I can look again to be sure.